src/core/Domino.qml

```diff
@@ -76,16 +82,53 @@ Flipable {
         border.width: flipable.borderWidth
         radius: flipable.radius
 
-        DominoNumber {
-            id: number1
+        DominoDot {
```

Comment:
I don't like the idea of triplicate the dominoes in the code + having 3 different dominoes with a lot of copy/paste.
Is there a way to simplify this?
Here there are dependencies between the dominoes (onValueChanged), and it seems except "visible" property, almost all the other property are copy/pasted for 6 items